src/dotnet/commands/dotnet-build/CompilerIOManager.cs

```diff
@@ -92,27 +93,28 @@ private CompilerIO ComputeIO(ProjectGraphNode graphNode)
             return new CompilerIO(inputs, outputs);
         }
 
-        private static void AddLockFile(ProjectContext project, List<string> inputs)
+        private static void AddLockFile(Project project, List<string> inputs)
         {
-            if (project.LockFile == null)
-            {
-                var errorMessage = $"Project {project.ProjectName()} does not have a lock file.";
-                Reporter.Error.WriteLine(errorMessage);
-                throw new InvalidOperationException(errorMessage);
-            }
-
-            inputs.Add(project.LockFile.LockFilePath);
-
-            if (project.LockFile.ExportFile != null)
-            {
-                inputs.Add(project.LockFile.ExportFile.ExportFilePath);
-            }
+            //TODO: better way to fix this:
+            //if (project.LockFile == null)
+            //{
+            //    var errorMessage = $"Project {project.ProjectName()} does not have a lock file.";
+            //    Reporter.Error.WriteLine(errorMessage);
+            //    throw new InvalidOperationException(errorMessage);
+            //}
+
+            //inputs.Add(project.LockFile.LockFilePath);
+
+            //if (project.LockFile.ExportFile != null)
```

Comment:
I don't know how to fix `ExportFile` how does it get generated? Can we try and hardcode a name somehow?